firebase-database/baseline.txt

```diff
@@ -0,0 +1,9 @@
+// Baseline format: 1.0
+RemovedInterface: com.google.firebase.database.Exclude:
```

Comment:
I don't understand these lines, all of these are annotations so clearly they implement `Annotation`, also what does it compare it to? Removed when?